packages/catalog-process/src/utilities/errorMappers.ts

```diff
@@ -44,7 +44,16 @@ export const createEServiceInstanceFromTemplateErrorMapper = (
       "eServiceNameDuplicate",
       () => HTTP_STATUS_CONFLICT
     )
-    .with("originNotCompliant", () => HTTP_STATUS_FORBIDDEN)
+    .with(
+      "originNotCompliant",
+      "receiveTemplateMissingTenantKindRiskAnalysis",
```

Comment:
It's not related to some permission but wrong eservice state, `HTTP_STATUS_CONFLICT` could be more appropriate.